lib/srv/db/opensearch/engine.go

```diff
@@ -332,21 +338,21 @@ func (e *Engine) emitAuditEvent(req *http.Request, body []byte, statusCode uint3
 
 // sendResponse sends the response back to the OpenSearch client.
 func (e *Engine) sendResponse(serverResponse *http.Response) error {
-	payload, err := utils.GetAndReplaceResponseBody(serverResponse)
-	if err != nil {
-		return trace.Wrap(err)
-	}
-
 	// serverResponse may be HTTP2 response, but we should reply with HTTP 1.1
 	clientResponse := &http.Response{
 		ProtoMajor:    1,
 		ProtoMinor:    1,
 		StatusCode:    serverResponse.StatusCode,
-		Body:          io.NopCloser(bytes.NewBuffer(payload)),
+		Body:          serverResponse.Body,
```

Comment:
I think this should be fine. I don't see why we should buffer the body before sending it to the client.